etc/kayobe/ansible/wazuh-manager.yml

```diff
@@ -26,66 +31,69 @@
             - (playbook_dir ~ '/wazuh/custom_certificates') is exists
 
         - name: Check that removed variable, local_custom_certs_path, is not set
-          assert:
+          ansible.builtin.assert:
             that: local_custom_certs_path is not defined
-            fail_msg: "The variable, `local_custom_certs_path`, is no longer used. Please remove this variable."
-      when:
-        - groups["wazuh-manager"] | length > 0
+            fail_msg: The variable, `local_custom_certs_path`, is no longer used. Please remove this variable.
 
-# Certificates generation
-- hosts: localhost
+- name: Apply Wazuh indexer role
```

Comment:
nit: Can we put the "Certificates generation" comment back? Even though we have ``generate-certs`` in tags, it'll be quicker to find what generates certificates with comment.

Reply:
Done, it got moved to the top of the file by the auto-linter for some reason